frontend/src/app/story-editor/story-editor.component.html

```diff
@@ -219,7 +219,8 @@ <h1 class="LargeHeadline2">You are not authorized to use this project</h1>
           <label class="selectLabel SmallIcontext">Browser:</label>
           <select name="browser" class="select" id="browserSelect" (change)="setBrowser($event, $event.target.value)" value="{{this.selectedScenario.browser || 'chrome'}}">
             <option *ngIf="this.chromium_enabled" value="chrome">Chrome</option>
-            <option *ngIf="this.gecko_enabled && !this.daisyVersion" value="firefox">Firefox</option>
+            <option  value="firefox">Firefox</option>
```

Comment:
Bitte wie bei Chrome mit einem ngIf versehen und die Steuerung, ob das angezeigt wird, in die server.js / .env auslagern:
CHROMIUM_ENABLED=true
GECKO_ENABLED=true
EDGE_ENABLED=true